docs/api.md

```diff
@@ -51,6 +51,7 @@
 .. autofunction:: scores.probability.murphy_thetas
 .. autofunction:: scores.probability.roc_curve_data
 .. autofunction:: scores.probability.brier_score
+.. autofunction:: scores.probability.ensemble_brier_score
```

Comment:
I don't see brier_score_for_ensemble in api.md. Has api.md been updated with the updated function name?

(I initially added this comment to a previously resolved conversation. In case GitHub continues to hide it, I thought I should open a new comment. So apologies for the repetition.)